code/script.js

```diff
@@ -1,51 +1,252 @@
 
 // Declare your functions after this comment
 
-// This function will add a chat bubble in the correct place based on who the sender is
+// This function will add a chat bubble in the correct place based on who the sender is. I think?
 const showMessage = (message, sender) => {
-  // the if statement checks if the sender is 'user' and if that's the case it inserts an html senction inside the chat with the posted message
+
   if (sender === 'user') {
     chat.innerHTML += `
       <section class="user-msg">
         <div class="bubble user-bubble">
           <p>${message}</p>
         </div>
-        <img src="assets/user.png" alt="User" />  
+        <img src="assets/user1.png" alt="User" />  
       </section>
     `
-    // the else if statement checks if the sender is a bot and if that's the case it inserts an html senction inside the chat with the posted message
-  } else if (sender === 'bot') {
+  }
+  else if (sender === 'bot') {
+    console.log(sender);
+    console.log(message);
     chat.innerHTML += `
       <section class="bot-msg">
-        <img src="assets/bot.png" alt="Bot" />
+        <img src="assets/bot1.png" alt="Bot" />
         <div class="bubble bot-bubble">
           <p>${message}</p>
         </div>
-      </section>
-    `
+      </section>`
   }
   // This little thing makes the chat scroll to the last message when there are too many to be shown in the chat box
   chat.scrollTop = chat.scrollHeight
 }
 
 // Starts here
+// here we call the function showMessage, that we declared earlier with the argument "Hello there, What's your name?" for message, and the argument "bot" for sender
+
 const greetUser = () => {
-  // here we call the function showMessage, that we declared earlier with the argument "Hello there, What's your name?" for message, and the argument "bot" for sender
-  showMessage("Hello there, What's your name?", 'bot')
-  // Just to check it out, change 'bot' to 'user' here 👆
+  showMessage(`Hello there!`, 'bot')
+  showMessage(`I'm Anna, a chatbot.`, 'bot')
+  showMessage(`What's your name?`, 'bot')
+}
+
+const userNameInput = event => {
+  event.preventDefault();
+  //I think the name is saved for later, but is it saved globally?
+  userName = nameInput.value;
+  nameInput.value = "";
+  showMessage(` ${userName}`, "user");
+  setTimeout(() => reply(userName), 500);
+  setTimeout(() => askForChoise(), 1200);
+};
+
+
+//Bot's welcome message with username pops up. Asks user to make a choise.
+function reply(firstReply) {
+  showMessage(`Welcome to 'Elbines-Plants' ${userName}!`, 'bot')
+}
+
+//I wanted a delay between the messages, thats why they are in two functions
+const askForChoise = (YesNo) => {
+  //Do I need to type anything in ()?
+  showMessage(`We can help you by selecting some options for you based on your preferences. <br><br> Would you like to continue?`, 'bot')
+  //Buttons Yes No  will appear
+
+  inputWrapper.innerHTML = //(When) shoud i use value (or class) instead of ID?
```

Comment:
I think it depends on which HTML element you use. The first one is about if you need a value for a select element. And the second link is about how value attribute works differently with different HTML elements.
Here is a link you might be interested in: https://stackoverflow.com/questions/38854683/is-a-value-attribute-required-for-a-html-select-element
This could be interesting too: https://www.geeksforgeeks.org/html-value-attribute/